src/generator.cs

```diff
@@ -5552,6 +5555,7 @@ void PrintExport (MemberInformation minfo, string sel, ArgumentSemantic semantic
 
 	void GenerateMethod (MemberInformation minfo)
 	{
+		using var span = GeneratorContext.BeginSpan ($"Method: {minfo.method.Name}");
```

Comment:
One option is to pass the string and a variable number of arguments, and only smash them together in a string if we're called to walk the stack.

I discarded it due to complexity for now. I could write up a performance benchmark and see how much it helps, but went with the simple and stupid thing for a RFC PR.

Reply:
Some actual numbers would be nice, maybe a timing of how long it takes to build src/, since that's the biggest test case we have.

Reply:
I ran `make clean && time make` on changes before and after the PR and got:

With changes: `make  1917.08s user 120.74s system 283% cpu 11:59.57 total`
Without changes: `make  1890.59s user 120.21s system 271% cpu 12:19.96 total`

So there is a noticeable slow down (27 seconds, 1.4%) of the world. 

However, given the massive single threaded build time, jitter seems possible to throw that off a bunch. I repeated just touching appkit.cs after a full builds.

`touch appkit.cs && time make`
With Changes: `make  767.57s user 38.25s system 307% cpu 4:22.30 total`
Without Changes: `make  752.03s user 37.04s system 294% cpu 4:27.58 total`
Roughly 2% slower.

Reply:
Now the question is if potential improvements by this RFC are worth spending times to optimize that away, if we care enough about 2% to do it, or if this isn't worth the time eitherway.